xerosdk/apis/contacts.py

```diff
@@ -35,3 +36,15 @@ def post(self, data):
         """
 
         return self._post_request(data, Contacts.POST_CONTACTS)
+
+
+    def search_contact_by_display_name(self,display_name: str):
+        """
+        Search contact by display name
+        :param display_name: Xero Contact Name
+        :return: Contact
+        """
+
+        response = self._get_request(Contacts.SEARCH_CONTACT.format(display_name))
```

Comment:
can return this directly, no use in storing this in a variable.
`{'Id': 'c9430a07-0319-4193-a65a-4fe7eb2fa170', 'Status': 'OK', 'ProviderName': 'Fyle Staging', 'DateTimeUTC': '/Date(1617805239991)/',` these details are not required for the API. Can return only the contact detail if it is present else None.

Reply:
okay

Reply:
`response = self._get_request(Contacts.SEARCH_CONTACT.format(display_name))

        return response['Contacts'][0] if len(response['Contacts']) > 0 else None`

Reply:
is this okay ??